src/domain/community/organization/organization.resolver.fields.ts

```diff
@@ -10,58 +10,98 @@ import { IOrganization } from '@domain/community/organization';
 import { IUserGroup } from '@domain/community/user-group';
 import { IUser } from '@domain/community/user';
 import { IProfile } from '@domain/community/profile';
-import { AuthorizationAgentPrivilege, Profiling } from '@common/decorators';
+import { CurrentUser, Profiling } from '@common/decorators';
 import { IAgent } from '@domain/agent/agent';
 import { UUID } from '@domain/common/scalars';
 import { UserGroupService } from '@domain/community/user-group/user-group.service';
 import { IOrganizationVerification } from '../organization-verification/organization.verification.interface';
 import { INVP } from '@domain/common/nvp/nvp.interface';
 import { IPreference } from '@domain/common/preference';
 import { PreferenceSetService } from '@domain/common/preference-set/preference.set.service';
+import { AgentInfo } from '@src/core';
+import { AuthorizationService } from '@core/authorization/authorization.service';
 @Resolver(() => IOrganization)
 export class OrganizationResolverFields {
   constructor(
+    private authorizationService: AuthorizationService,
     private organizationService: OrganizationService,
     private groupService: UserGroupService,
     private preferenceSetService: PreferenceSetService
   ) {}
 
-  @AuthorizationAgentPrivilege(AuthorizationPrivilege.READ)
+  //@AuthorizationAgentPrivilege(AuthorizationPrivilege.READ)
```

Comment:
Revert the changes re AuthorizationAgentPrivilege. Using the decorator builds & executes a rule and I don't think it is necessary to have if ... this.isAccessGranted ... etc.